core/frontend/src/tile/map/ImageryProviders/WmtsMapLayerImageryProvider.ts

```diff
@@ -96,7 +96,7 @@ export class WmtsMapLayerImageryProvider extends MapLayerImageryProvider {
       }
 
       if (preferredTms !== undefined) {
-        const tmsLink= layer.tileMatrixSetLinks.find((tmsl) => tmsl.tileMatrixSet === preferredTms!.identifier);
```

Comment:
Wondering what could be done here to avoid the non-null assertion operator ?

Reply:
Maybe I'm misunderstanding but isn't the non null assertion operator gone now?

Reply:
haha you are right, sorry my brain was upside down this morning :-)